tests/server/TestFrameGuardConfig.js

```diff
@@ -1,16 +1,22 @@
 import { assert } from 'chai';
 import requireUncached from 'require-uncached';
+import config from 'config';
+
+const appsList = config.get('validAppNames');
 
 describe('App Specific Frameguard Config', () => {
   afterEach(() => {
     process.env.NODE_ENV = 'production';
     delete process.env.NODE_APP_INSTANCE;
   });
 
-  it('should default frameGuard to "deny"', () => {
-    const config = requireUncached('config');
-    const frameGuardConfig = config.get('frameGuard');
-    assert.equal(frameGuardConfig.action, 'deny');
-    assert.equal(frameGuardConfig.domain, undefined);
-  });
+  for (const appName of appsList) {
```

Comment:
What's the reason for `const` over `let` in these loops? I suppose the app names should be immutable. Just threw me off. I suppose I answered my own question 😁